Formula/wcslib.rb

```diff
@@ -0,0 +1,30 @@
+class Wcslib < Formula
+  desc "Library and utilities for the FITS World Coordinate System"
+  homepage "https://www.atnf.csiro.au/people/mcalabre/WCS/"
+  url "ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-5.19.1.tar.bz2"
+  sha256 "59b9f0e5a2c040773cc846c684d84c09b986c1393e97b378a41b92d9d3df0f98"
+
+  depends_on "cfitsio"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--with-cfitsiolib=#{Formula["cfitsio"].opt_lib}",
+                          "--with-cfitsioinc=#{Formula["cfitsio"].opt_include}",
+                          "--without-pgplot",
+                          "--disable-fortran"
+
+    # Currently doesn't support parallel make.  Patch sent to author 2018/08/31.
+    # Author expects to integrate end of 2018/09.
```

Comment:
Can we add the Author's email here? Maybe:

```
# Author (mcalabre@atnf.csiro.au) expects to integrate end of 2018/09.
```

(The author publishes his email without obfuscation on the package homepage... otherwise we could say mcalabre (at) atnf.csiro.au etc, but I'm inclined not to, since the email is already public on the upstream homepage.)